LMS/Landing/Resources/views/welcome.blade.php

```diff
@@ -0,0 +1,201 @@
+                        allowfullscreen></iframe>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<div class="container">
+    <section id="trophys" class="text-center">
+        <div class="row justify-content-center">
+            <div class="col-12 col-md-8">
+                <h2 class="title">{{ trans('landing::view.achievements.title') }}</h2>
+                <h5 class="description">{{ trans('landing::view.achievements.lead') }}</h5>
+            </div>
+        </div>
+
+        <div class="features">
+            <div class="row">
+                <div class="col-md-4">
+                    <div class="info">
+                        <div class="icon icon-info">
+                            <i class="material-icons">school</i>
+                        </div>
+                        <h4 class="info-title">
+                            <span style="font-weight: bold">{{ $landingData['achievements']['students'] }}</span>
+                            {{ trans('landing::view.achievements.items.students') }}
+                        </h4>
+                    </div>
+                </div>
+
+                <div class="col-md-4">
+                    <div class="info">
+                        <div class="icon icon-success">
+                            <i class="material-icons">videocam</i>
+                        </div>
+                        <h4 class="info-title">
+                            <span style="font-weight: bold">{{ $landingData['achievements']['contentHours'] }}h</span>
+                            {{ trans('landing::view.achievements.items.contentTime') }}
+                        </h4>
+                    </div>
+                </div>
+
+                <div class="col-md-4">
+                    <div class="info">
+                        <div class="icon icon-danger">
+                            <i class="material-icons">cast_for_education</i>
+                        </div>
+                        <h4 class="info-title">
+                            <span style="font-weight: bold">{{ $landingData['achievements']['watchedLessons'] }}</span>
+                            {{ trans('landing::view.achievements.items.watchedLessons') }}
+                        </h4>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <hr>
+    <section id="currentCourses" class="section text-center">
+        <h2 class="title">{{ trans('landing::view.availableCourses.title') }}</h2>
+        <div class="row justify-content-center">
+            @foreach($courses = \LMS\Courses\Models\Course::paginate() as $course)
+                <div class="col-12 col-sm-6 col-md-4">
+
+                    <div class="card card-product">
+                        <div class="card-header card-header-image">
+                            <img class="img" src="{{ $course->getFirstMediaUrl() }}">
```

Comment:
Para melhor acessibilidade inserir `alt` nas tag de `<img />`

```suggestion
                            <img class="img" src="{{ $course->getFirstMediaUrl() }}" alt="Descreva a imagem de acordo com o que você está vendo, daniel">
```

Reply:
Corrigido <3